inst/stan/epinowcast.stan

```diff
@@ -39,6 +35,11 @@ data {
   matrix[urds, nrd_effs + 1] rd_fixed; // design matrix for report dates
   int nrd_eff_sds; // number of standard deviations to use for pooling for rds
   matrix[nrd_effs, nrd_eff_sds + 1] rd_random; // Pooling pmf design matrix 
+  // Observations
+  int obs[s, dmax]; // obs for each primary date (row) and report date (column)
+  int flat_obs[n]; // obs stored as a flat vector
+  int obs_miss[g, t]; // obs with missing primary date (group first)
```

Comment:
I put group first for reasons of memory locality. The time is used in a vectorized statement in the likelihood and is therefore the fast index, the groups are outer looped over and are therefore the slow index. Not sure if we want to change latest_obs as well for for consistency: the current order for latent obs is actually the more efficient one because there group is the fast index, on the other hand it is only used in the transformed data anyway.

Reply:
Yeah I think changing this so they are all the same makes sense?